Annotation/polar_golden_set.json

```diff
@@ -634,13 +695,13 @@
             {
                 "predicate": [
                     {
-                        "str": "known",
+                        "str": "known about",
                         "lemma": "",
                         "POS": ""
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "whether there is any information connecting Spedding to this cabin",

```

Comment:
The way I interpret it, the clause is a complement of "little" and not of the verb which would maybe make it not an embedded clause at all. The active version is "x knows little about this and whether there is any information...". So, "little about this" and "little about whether there is any information..." is known

Reply:
Just to be clear, is Sivan Milton arguing that the little more is embedding, and thus "little more" has thus **moved** position in the sentence??? So, personally I think we should keep, because it's acting as an NPI and we would include clauses like that, e.g. "John knows not whether Mary likes mark" - old fashioned I know, but I think here we would argue its acting as an NPI and its part of a more complicated predicate

Reply:
Hey I neeever said the passive was derived syntactically from the active via movement. I'm just looking at it semantically here. But I'm cool with us keeping this in - only we should maybe keep it in some list of weird cases in the Monster Doc